samples/display/cfb_shell/README.rst

```diff
@@ -0,0 +1,75 @@
+.. _cfb_shell_sample:
+
+Character Framebuffer Shell Module Sample
+#########################################
+
+Overview
+********
+This is a simple shell module that exercises displays using the Character
+Framebuffer subsystem.
+
+Building and Running
+********************
+
+Build the sample app by choosing the OVERLAY_CONFIG for the target board, for
+example:
+
+.. code-block:: console
+
+         cmake -DBOARD=reel_board
+
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/display/cfb_shell
+   :goals: run
+
+
+Shell Module Command Help
+=========================
+
+.. code-block:: console
+
+         cfb - Character Framebuffer shell commands
+         Options:
+                 -h, --help  :Show command help.
+         Subcommands:
+                 init        :[none]
+                 get_device  :[none]
+                 get_param   :<all, height, width, ppt, rows, cols>
+                 get_fonts   :[none]
+                 set_font    :<idx>
+                 invert      :[none]
+                 print       :<col: pos> <row: pos> <text>
+                 scroll      :<dir: (vertical|horizontal)> <col: pos> <row: pos>
+                              <text>
+                 clear       :[none]
+
+**init**: should be called first to initialize the display.
+
+**get_device**: prints the display device name.
+
+**get_param**: get the display parameters where height, width and ppt
+(pixel per tile) are in pixels and the number of rows and columns. The row
+position is incremented by a multiple of the ppt.
+
+**get_fonts**: print the index, height and width in pixels of the static
+defined fonts presented in the system.
+
+**set_font**: choose the font to be used by passing the font index. Only one
+font can be used at a time.
+
+**invert**: invert the pixel color of the display.
+
+**print**: pass the initial column and row positions and the text in
+double quotation marks when it contains spaces. If text hits the edge
+the display the remain characters will be displayed in the next line. The
+previous printed text will be erased.
+
+**scroll**: pass the scroll direction, vertical or horizontal, the initial
+column and row positions, and the text in double quotation marks when it
+contains spaces. If text hits the edge the display the remaining characters
+will be displayed in the next line. The text will scroll until it hits the
+display boundary, last column for horizontal and last row for vertical
+direction.
+
+**clear**: clear the display screen.
```

Comment:
It would be great, after these descriptions, to show a nice example of how to display a phrase on the display and manipulate it (and have pictures or describe what gets displayed).

Reply:
I can put command examples as @nashif suggested, but at the moment there is no API, AFAIK, to draw pictures, or manipulated what is printed like inversion.

Reply:
@diegosueiro I think what @dbkinder means is to give examples with screenshots in the docs.
But on adding the screenshots in the docs I already raised the question (PR #9149) if it is allowed to add binary files into the git repo. In my opinion this is not a good idea and git-lfs is not setup for the zephyr repo (@nashif any inputs on this?)

Regarding the API for drawing you have PR #6826 if you need something more fancy than CFB. CFB is just a lightweight framework for displaying text where LVGL is a full blown GUI library and I think they complement each other nicely